freeclimb-cs-sdk-test/webhooks/percl/GetSpeechActionCallbackTest.cs

```diff
@@ -20,11 +20,15 @@ public void CreateAndVerifyGetSpeechStatusCallbackDigitReasonTest()
         [TestMethod]
         public void CreateAndVerifyGetSpeechStatusCallbackDigitRecognitionTest()
         {
-            GetSpeechActionCallback getSpeechStatusCallback = GetSpeechActionCallback.fromJson("{\"accountId\":\"ACe1006ad515dbfc486dcb75d08db1445b928aef08\",\"callId\":\"CA2a98e2a2635c7cbd610d1085099a6c73e8990c7a\",\"callStatus\":\"inProgress\",\"conferenceId\":null,\"confidence\":92,\"direction\":\"outboundAPI\",\"from\":\"+12248806205\",\"parentCallId\":null,\"queueId\":null,\"reason\":\"recognition\",\"requestId\":\"RQc6a43de7b417b7c77b6a1e3c27628bb83c86c6ca\",\"requestType\":\"getSpeech\",\"recognitionResult\":\"3 3 3\",\"to\":\"+18475978014\"}");
+            GetSpeechActionCallback getSpeechStatusCallback = GetSpeechActionCallback.fromJson("{\"accountId\":\"ACe1006ad515dbfc486dcb75d08db1445b928aef08\",\"callId\":\"CA2a98e2a2635c7cbd610d1085099a6c73e8990c7a\",\"callStatus\":\"inProgress\",\"conferenceId\":null,\"confidence\":92,\"direction\":\"outboundAPI\",\"from\":\"+12248806205\",\"parentCallId\":null,\"queueId\":null,\"reason\":\"recognition\",\"requestId\":\"RQc6a43de7b417b7c77b6a1e3c27628bb83c86c6ca\",\"requestType\":\"getSpeech\",\"recognitionResult\":\"3 3 3\",\"completionReason\":\"test\",\"completionCause\":\"test\",\"mrcpCode\":123,\"mrcpDiagnostic\":\"test\",\"to\":\"+18475978014\"}");
```

Comment:
C# has multi-line strings capabilities, although not the greatest when it comes to JSON strings might make this a little easier to read.
```
.fromJson(@"
    ""accountId"": ""..."",
    ""callId": ""...""
")
```

Reply:
Created a ticket to address this